### PR TITLE
completed express-post-json | Cody Wiseman

### PR DESCRIPTION
**Pull request by codywiseman**
Screen Shot 2021-01-07 at 12 54 21 PM

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (let id in grades) {
+    gradesArray.push(grades[id]);
+  }
+  res.json(gradesArray);
+})
+
+app.use(express.json());
```
Comment (BrettAlbright) [inline express-post-json/index.js]: this needs to be above all of the endpoints

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small change needed

**Review by BrettAlbright — APPROVED** [review]
👍

File: express-post-json/index.js (hunk 1 above)
Comment (codywiseman) [reply express-post-json/index.js]: Screen Shot 2021-01-07 at 1 05 01 PM

**Pull request merged**